From 51223532c7771770f6204749267861b68ed55ae6 Mon Sep 17 00:00:00 2001
From: Raghavendra Talur <rtalur@redhat.com>
Date: Mon, 18 Jan 2016 21:51:41 +0530
Subject: [PATCH 247/251] glusterd: define GD_OP_VERSION_MAX

The patch which defined GD_OP_VERSION_MAX isn't
pulled/required downstream but we need this definition
for write-behind patch.[1]

[1]: https://code.engineering.redhat.com/gerrit/#/c/65752/

Change-Id: Idb2714c0822acbf04173eb204cf8f53272c419c7
BUG: 1219794
Signed-off-by: Raghavendra Talur <rtalur@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/65818
Reviewed-by: Raghavendra Gowdappa <rgowdapp@redhat.com>
Tested-by: Raghavendra Gowdappa <rgowdapp@redhat.com>
---
 libglusterfs/src/globals.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/libglusterfs/src/globals.h b/libglusterfs/src/globals.h
index 7eaa646..cc23b6c 100644
--- a/libglusterfs/src/globals.h
+++ b/libglusterfs/src/globals.h
@@ -42,7 +42,7 @@
  */
 #define GD_OP_VERSION_MIN  1 /* MIN is the fresh start op-version, mostly
                                 should not change */
-#define GD_OP_VERSION_MAX  GD_OP_VERSION_3_7_6 /* MAX VERSION is the maximum
+#define GD_OP_VERSION_MAX  GD_OP_VERSION_3_7_7 /* MAX VERSION is the maximum
                                                   count in VME table, should
                                                   keep changing with
                                                   introduction of newer
@@ -70,6 +70,8 @@
 
 #define GD_OP_VERSION_3_7_6    30706 /* Op-version for GlusterFS 3.7.6 */
 
+#define GD_OP_VERSION_3_7_7    30707 /* Op-version for GlusterFS 3.7.7 */
+
 #include "xlator.h"
 
 /* THIS */
-- 
1.7.1

